modules/optflow/src/variational_refinement.cpp

```diff
@@ -1102,8 +1102,8 @@ void VariationalRefinementImpl::calcUV(InputArray I0, InputArray I1, InputOutput
     int num_stripes = getNumThreads();
     Mat I0Mat = I0.getMat();
     Mat I1Mat = I1.getMat();
-    Mat &W_u = flow_u.getMatRef();
-    Mat &W_v = flow_v.getMatRef();
+    Mat W_u = flow_u.getMat();
+    Mat W_v = flow_v.getMat();
```

Comment:
In case of this change you should update OutputArray in the end of this function:
```
flow_u.assign(W_u);
flow_v.assign(W_v);
```